ckanext/collaborators/plugin.py

```diff
@@ -45,3 +52,26 @@ def get_auth_functions(self):
             'dataset_collaborator_delete': auth.dataset_collaborator_delete,
             'dataset_collaborator_list': auth.dataset_collaborator_list,
         }
+
+    # ITemplateHelpers
+    def get_helpers(self):
+        return {'collaborators_get_collaborators': get_collaborators}
+
+    # IBlueprint
+    def get_blueprint(self):
+        blueprint = Blueprint(self.name, self.__module__)
```

Comment:
I prefer the convention of having a `ckanext/collaborators/blueprint.py` module defining the blueprint (named `collaborators`) and then from `plugin.py` do

```python
from ckanext.collaborators import blueprint

class CollaboratorsPlugin(p.SingletonPlugin):

      # ....
      def get_blueprint(self):
           return blueprint.collaborators

```